src/main/java/io/cdap/plugin/gcp/spanner/sink/SpannerSinkConfig.java

```diff
@@ -147,6 +148,12 @@ public String getKeys() {
     return keys;
   }
 
+  public String getReferenceNameOrNoarmalizedFQN() {
+    return Strings.isNullOrEmpty(referenceName)
+      ? ReferenceNames.normalizeFqn(getFQN())
```

Comment:
nit - can you put this on the same line?

Reply:
+1